frappe/custom/doctype/customize_form/customize_form.py

```diff
@@ -68,7 +68,7 @@
 }
 
 allowed_fieldtype_change = (('Currency', 'Float', 'Percent'), ('Small Text', 'Data'),
-	('Text', 'Data'), ('Text', 'Text Editor', 'Code', 'Signature', 'HTML Editor'), ('Data', 'Select'),
+	('Text', 'Data'), ('Text', 'Text Editor', 'Code', 'Signature', 'HTML Editor', 'Markdown Editor'), ('Data', 'Select'),
```

Comment:
I don't think fields with values as HTML and Markdown are interchangeable.

Reply:
You told me to put this here. Should I remove this then?

Reply:
Yeah, remove it.